rllib/algorithms/dreamer/dreamer.py

```diff
@@ -118,6 +119,11 @@ def __init__(self):
             # Repeats action send by policy for frame_skip times in env
             "frame_skip": 2,
         })
+
+        # .exploration()
+        # This dreamer implementation does not need an exploration config
+        self.exploration_config = None
```

Comment:
is there a difference between empty dict or None here? For consistency I'd like to use `{}` if possible.

Reply:
Altough it won't make a difference in tests, we have some code that assumes that exploration config is always a dict, so you are right.